en/04_Changelogs/5.2.0.md

```diff
@@ -87,7 +88,13 @@ SilverStripe\ErrorPage\ErrorPage:
     - 500
 ```
 
-### Other new features
```

Comment:
Don't remove this heading - we'll need this for other changes, and it's still referenced in the table of contents.

Reply:
Added back in